xbmc/utils/SortUtils.cpp

```diff
@@ -276,6 +276,22 @@ std::string BySortTitle(SortAttribute attributes, const SortItem &values)
   return title;
 }
 
+std::string ByOriginalTitle(SortAttribute attributes, const SortItem& values)
+{
+
+  std::string title = values.at(FieldOriginalTitle).asString();
+  if (title.empty())
+    title = values.at(FieldSortTitle).asString();
+
+  if (title.empty())
+    title = values.at(FieldTitle).asString();
+
+  if (attributes & SortAttributeIgnoreArticle)
```

Comment:
Maybe `if (!title.empty() && (attributes & ...))` ?

Reply:
Are you sure that _SortAttributeIgnoreArticle_ should be dependent on non-empty title? In that case I think _std::string BySortTitle_ must be changed also for consistency

Reply:
what could the result of `SortUtils::RemoveArticles` called with an empty string be? nothing else tan an ampty string, right? So, if title is empty, there should be no need to call `SortUtils::RemoveArticles`. That's the whole idea.

Reply:
I don't think `title` will ever be empty because we have the fallback to use `FieldTitle` and I don't think many people have items without titles in their libraries.

Reply:
Okay, then, Thanks for clarifying, @Montellese